docs/elements.md

```diff
@@ -2026,7 +2026,7 @@ Common reasons that trigger the display of the answer panel are:
 
 ### `pl-external-grader-results` element
 
-Displays results from externally-graded questions.
+This element displays results from externally-graded questions.
```

Comment:
Dubious fix, though not sure it matters too much.

Reply:
Why change this instance but none other in this file? Let's be consistent; I'd err on the side of standardizing on no "this element".

Reply:
e6b0639e7c88886a24ccbe44ccea193fd539369f